pkgs/development/python-modules/numpy-groupies/default.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pytestCheckHook,
+  setuptools,
+  setuptools-scm,
+  numpy,
+  numba,
+  pandas,
+  gitUpdater,
+}:
+
+buildPythonPackage rec {
+  pname = "numpy-groupies";
+  version = "0.11.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "ml31415";
+    repo = "numpy-groupies";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-Eu+5SR28jIasKe1p7rvbq2yo3PGZRQWWdG3A5vGhnyM=";
+  };
+
+  build-system = [
+    setuptools
+    setuptools-scm
+  ];
+
+  dependencies = [ numpy ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    numba
+    pandas
+  ];
+
+  pythonImportsCheck = [ "numpy_groupies" ];
+
+  passthru.updateScript = gitUpdater { tagPrefix = "v"; };
```

Comment:
This update script doesn't eval, which broke part of the `nixpkgs-update` bot that tries to run every updateScript.

```
$ nix-shell maintainers/scripts/update.nix --argstr package python3Packages.numpy-groupies
error:
       … while calling the 'derivationStrict' builtin

         at /builtin/derivation.nix:9:12: (source not available)

       … while evaluating derivation 'nixpkgs-update-script'
         whose name attribute is located at /home/zowoq/nixpkgs/pkgs/stdenv/generic/make-derivation.nix:334:7

       … while evaluating attribute 'shellHook' of derivation 'nixpkgs-update-script'

         at /home/zowoq/nixpkgs/maintainers/scripts/update.nix:229:3:

          228|   '';
          229|   shellHook = ''
             |   ^
          230|     unset shellHook # do not contaminate nested shells

       error: function 'anonymous lambda' called with unexpected argument 'tagPrefix'

       at /home/zowoq/nixpkgs/pkgs/common-updater/git-updater.nix:6:1:

            5|
            6| { pname ? null
             | ^
            7| , version ? null

```

Reply:
Sorry for the breakage.
I've fixed it by https://github.com/NixOS/nixpkgs/pull/334084, thank you.